diff -urN 2ManDVD/videowrapper.cpp 2ManDVD-patched/videowrapper.cpp
--- 2ManDVD/videowrapper.cpp	2013-07-30 22:46:19.000000000 +1100
+++ 2ManDVD-patched/videowrapper.cpp	2013-08-09 22:20:49.000000000 +1100
@@ -38,13 +38,8 @@
 int videowrapper::load_video(QString filename)
 {
     // *** OUVERTURE DU FICHIER VIDEO ***
-    #ifdef avformat_open_input
         if (avformat_open_input(&pFormatCtx,filename.toAscii().data(),NULL,NULL)!=0)
         {
-    #else
-        if (av_open_input_file(&pFormatCtx,filename.toAscii().data(),NULL,0,NULL)!=0)
-        {
-    #endif
         qDebug() << "CAN'T OPEN VIDEO FILE : " << filename;
         return 1;
     }
@@ -91,13 +86,8 @@
 
     // *** OUVERTURE DU CODEC ***
 
-    #ifdef av_find_stream_info
-    if(avcodec_open(pCodecCtx, pCodec)<0)
-    {
-    #else
     if (avcodec_open2(pCodecCtx, pCodec, NULL) < 0)
     {
-    #endif
         qDebug() << "CAN'T FIND VIDEO CODEC";
         return 1;
     }
